### PR TITLE
MDCT-291: Part 2 - layout folder tests

### PR DESCRIPTION
**Pull request by BearHanded**
# Description

This is all the tests for one folder.

## How to test

run tests

## Dependencies

Nothing new

# Get it done

Numbers go up

## Code authors checklist

- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [ ] Necessary analytics were added, or no new analytics were required
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

File: services/ui-src/src/components/layout/CertifyAndSubmit.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const Submit = ({ certify }) => (
       compliance with Title XXI of the Social Security Act (Section 2109(a) and
       Section 2108(e)).
     </p>
-    <Button onClick={certify} variation="primary">
+    <Button data-testid={"certifySubmit"} onClick={certify} variation="primary">
```
Comment (braxex) [inline services/ui-src/src/components/layout/CertifyAndSubmit.js:30]: Nit: Don't need brackets.

Comment (BearHanded) [reply services/ui-src/src/components/layout/CertifyAndSubmit.js:30]: agreed

File: services/ui-src/src/components/layout/CertifyAndSubmit.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  it("should display thanks after submit", () => {
+    render(submitFinished);
+    const thankYouElement = screen.getByTestId("certifyThankYou");
+    expect(thankYouElement).toHaveTextContent("Thank you");
+  });
+  it("should display submit button", () => {
+    render(submit);
+    const thankYouElement = screen.getByTestId("certifySubmit");
+    expect(thankYouElement).toHaveTextContent("Certify and Submit");
+  });
```
Comment (braxex) [inline services/ui-src/src/components/layout/CertifyAndSubmit.test.js:76]: Nit: My brain wants these to be swapped.

File: services/ui-src/src/components/layout/FormTemplates.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { shallow } from "enzyme";
+import FormTemplates from "./FormTemplates";
+import { act, render, fireEvent } from "@testing-library/react";
+
+const mockPost = jest.fn();
+jest.mock("aws-amplify", () => ({
+  __esModule: true,
```
Comment (gmrabian) [inline services/ui-src/src/components/layout/FormTemplates.test.js:8]: nit: don't need

File: services/ui-src/src/components/layout/Section.test.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+const testSectionId = "2020-123";
+const testSubSectionId = "1";
+const store = mockStore({
+  stateUser: {
+    name: "Kentucky",
+    imageURI: "kentucky.png",
+  },
+});
+jest.mock("../../store/selectors", () => ({
+  __esModule: true,
```
Comment (gmrabian) [inline services/ui-src/src/components/layout/Section.test.js:18]: nit: don't need `__esModule: true`

**Review by gmrabian — DISMISSED** [review]

**Review by karla-vm — DISMISSED** [review]

**Review by gmrabian — DISMISSED** [review]

**Review by braxex — DISMISSED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Review by braxex — APPROVED** [review]

**Pull request merged**